department-of-veterans-affairs/caseflow
# APPEALS-31187

Resolves [APPEALS-31187](https://jira.devops.va.gov/browse/APPEALS-31187)

# Description
Updates the change history table to use the backend data

## Acceptance Criteria
- [x] All controller methods for individual claim history view are updated to no longer return mock data, if required
- [ ] All front-end methods are updated to not use a mock request body, if required
- [x] Modify change history workflow to call appropriate service class methods and coordinate the flow of real data from request to response, if required 


## Testing Plan

1. Go to [Jira Issue/Test Plan Link](https://jira.devops.va.gov/browse/JIRA-12345) or list them below
2. Log in as VHAADMIN
3. Navigate to the decision review queue and click one
4. Click the view history button
5. verify that the claim history table now shows the data for that individual claim instead of dummy data


# Frontend
## User Facing Changes

<img width="1180" alt="image" src="https://github.com/department-of-veterans-affairs/caseflow/assets/17840614/022356f0-3acc-4cb8-97d5-4b5ddff74929">

## Storybook Story
*For Frontend (Presentation) Components*

# Backend
## Database Changes
*Only for Schema Changes*


## Integrations: Adding endpoints for external APIs

# Best practices
## Code Documentation Updates

## Tests
### Test Coverage
Did you include any test coverage for your code? Check below:

### Code Climate
Your code does not add any new code climate offenses? If so why?

## Monitoring, Logging, Auditing, Error, and Exception Handling Checklist

### Monitoring

### Logging

### Auditing

### Error Handling

### Exception Handling
